ntbs-service/Pages/ContactDetails/Edit.cshtml.cs

```diff
@@ -72,10 +72,6 @@ public async Task<IActionResult> OnPostAsync()
         private void ValidateModel()
         {
             TryValidateModel(ContactDetails, nameof(ContactDetails));
-            if (ContactDetails.ArePrimaryContactDetailsMissing)
-            {
-                ModelState.AddModelError("ContactDetails", ValidationMessages.SupplyCaseManagerPrimaryParameter);
```

Comment:
Presumably we can also delete the validation message, as I assume it's not being used elsewhere?

Reply:
Yep have deleted it now